meta-mender-qemu/recipes-support/gnutls/gnutls_%.bbappend

```diff
@@ -0,0 +1,3 @@
+
+PACKAGECONFIG_append = " p11-kit"
```

Comment:
This is correct. It's needed because gnutls is usually compiled without p11-kit support (this is support for p11-kit inside gnutls, completely independent of the package called p11-kit).

Reply:
I am no expert on this, this was an attempt to get p11tool built, I will go with your suggestions. I can try to remove and rebuilt and test.

Reply:
Thanks @kacf. Got it.